__tests__/workers/notifications.ts

```diff
@@ -2433,3 +2437,78 @@ describe('user post added', () => {
     expect(bundle.ctx.userIds).toIncludeSameMembers(['3', '4']);
   });
 });
+
+describe('plus subscription gift', () => {
+  type ObjectType = Partial<User>;
+  const base: ChangeObject<ObjectType> = {
+    ...usersFixture[0],
+    subscriptionFlags: JSON.stringify({}),
+  };
+  const plusUser = {
+    ...base,
+    subscriptionFlags: JSON.stringify({ cycle: SubscriptionCycles.Yearly }),
+  };
+  const giftedPlusUser = {
+    ...base,
+    subscriptionFlags: JSON.stringify({
+      cycle: SubscriptionCycles.Yearly,
+      gifterId: 2,
+      giftExpirationDate: addYears(new Date(), 1),
+    }),
+  };
+
+  beforeEach(async () => {
+    await saveFixtures(con, Source, [
+      {
+        id: PLUS_MEMBER_SQUAD_ID,
+        name: 'Plus Squad',
+        image: 'http://image.com/a',
+        handle: 'plus-squad-notify',
+        type: SourceType.Squad,
+      },
+    ]);
+  });
+
+  it('should early return for currently non plus user', async () => {
+    const worker = await import(
+      '../../src/workers/notifications/userGiftedPlusNotification'
+    );
+    const actual = await invokeNotificationWorker(worker.default, {
```

Comment:
Can you add this tests in a separate file as we did for moderation workers, we want to deprecate this giant notifications worker test file in the future and stop adding to it.